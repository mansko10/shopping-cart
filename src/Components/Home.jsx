import StyledHome from "./Styles/StyledHome.styled";
import { Link } from "react-router-dom";

export default function Home() {
  return (
    <StyledHome>
      <h1>Welcome to our shop!</h1>
      <Link to="/shop">Start Shopping</Link>
    </StyledHome>
  );
}
