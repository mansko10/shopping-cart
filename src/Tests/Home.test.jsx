import { it, expect } from "vitest";
import { render } from "@testing-library/react";
import Home from "../Components/Home";
import { BrowserRouter } from "react-router-dom";

it("Renders Home.jsx correctly", () => {
  const { container } = render(<Home />, { wrapper: BrowserRouter });

  expect(container).toMatchSnapshot();
});
