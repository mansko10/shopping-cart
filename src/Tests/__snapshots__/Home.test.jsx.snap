// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Renders Home.jsx correctly 1`] = `
<div>
  <div
    class="sc-blHHSb jNgcFa"
  >
    <h1>
      Welcome to our shop!
    </h1>
    <a
      data-discover="true"
      href="/shop"
    >
      Start Shopping
    </a>
  </div>
</div>
`;
